sdk/src/main/scala/com/horizen/SidechainNodeViewHolder.scala

```diff
@@ -295,6 +312,8 @@ object SidechainNodeViewHolder /*extends ScorexLogging with ScorexEncoding*/ {
     case class ApplyFunctionOnNodeView[HIS, MS, VL, MP, A](f: java.util.function.Function[SidechainNodeView, A])
     case class ApplyBiFunctionOnNodeView[HIS, MS, VL, MP, T, A](f: java.util.function.BiFunction[SidechainNodeView, T, A], functionParameter: T)
     case class LocallyGeneratedSecret[S <: SidechainTypes#SCS](secret: S)
+    case class GetStorageVersions()
```

Comment:
can be changed to `case object GetStorageVersions` since it doesn't require any parameters.